src/main/java/core/basesyntax/lib/Dao.java

```diff
@@ -4,5 +4,5 @@
 import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
-public @interface DaoAnnotation {
+public @interface Dao {
```

Comment:
missed that you have same name for interface and annotation, my bad 
in this case it would be better to have:
interface - Dao 
annotation - DaoImpl
but for now no need to refactor your code

Reply:
> missed that you have same name for interface and annotation, my bad
> in this case it would be better to have:
> interface - Dao
> annotation - DaoImpl
> but for now no need to refactor your code

got it, thanks! will keep in mind